repo/packages/C/confluent/3/resource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://s3.amazonaws.com/downloads.mesosphere.io/kafka/assets/jre-8u72-linux-x64.tar.gz",
```

Comment:
With the latest release we're now using an updated JRE package at `https://downloads.mesosphere.com/kafka/assets/jre-8u91-linux-x64.tar.gz`

The packages for the latest framework build can be found here:
`https://downloads.mesosphere.com/kafka/assets/1.0.8-0.10.0.0/scheduler.zip` and `https://downloads.mesosphere.com/kafka/assets/1.0.8-0.10.0.0/overrider.zip`

(see also: https://github.com/mesosphere/universe-1.7/blob/version-2.x/repo/packages/K/kafka/15/resource.json )